Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -109,84 +92,14 @@ class ShareExtensionHelper: NSObject {
                         completion(url, nil)
                     } catch {
                         completion(nil, error)
-                        log.error("Failed to write PDF to disk: \(error)")
+                        Logger.browserLogger.error("Failed to write PDF to disk: \(error)")
```

Comment:
No need to change those logger methods, this is code convention we use across our app.